scripts/tests/test_zkai_native_attention_mlp_rmsnorm_adjacent_layout_gate.py

```
@@ -0,0 +1,107 @@
+            with self.subTest(key=key):
+                payload = gate.build_payload()
+                payload[key] = value
+                payload["payload_commitment"] = gate.payload_commitment(payload)
+                with self.assertRaises(gate.AdjacentLayoutGateError):
+                    gate.validate_payload(payload)
+
+    def test_payload_rejects_non_claim_erasure(self) -> None:
+        payload = gate.build_payload()
+        payload["non_claims"] = []
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        with self.assertRaises(gate.AdjacentLayoutGateError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)